src/sharing/components/Recipient.jsx

```diff
@@ -166,7 +166,7 @@ const Recipient = (props, { client, t }) => {
   const { instance, isOwner, status, ...rest } = props
   const isMe =
     (isOwner && status === 'owner') || instance === client.options.uri
-  const name = Contact.getDisplayName(rest)
+  const name = t(Contact.getDisplayName(rest))
```

Comment:
really not a big fan of that.

Reply:
We could either : 
1- pass `t()` to `getDisplayName`
2- pass the default value (already translated) to `getDisplayName` when we need it (and use `''` as default)

2 seems ok but I would like to avoid to change every call to `Contact.getDisplayName` because this case where we have no name/email/public_name seems to append only in certains cases for recipients in "Shared with me" folder